client/components/social-icons/gmail.jsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { PureComponent } from 'react';
+
+import './style.scss';
+
+export default class GmailIcon extends PureComponent {
+	static propTypes = {
+		isDisabled: PropTypes.bool,
+	};
+
+	static defaultProps = {
+		isDisabled: false,
+	};
+
+	render() {
+		const { isDisabled, ...rest } = this.props;
+
+		return (
+			<svg
+				className={ classNames( 'social-icons social-icons__gmail', {
+					'social-icons--enabled': ! isDisabled,
+					'social-icons--disabled': !! isDisabled,
```

Comment:
Strange to have two classes? It should be default vs disabled.

Reply:
Yes, I have it copied from the existing components but seems nonsense. 👍